File: Practica 8/Problema2/Kotlin/Bird.kt
import AbstractAnimal

class Bird(name: String, year: Int) : AbstractAnimal(name, year, 3) {
    override fun move(): String {
        return "fly"
    }

    override fun breathe(): String {
        return "lungs"
    }

    override fun reproduce(): String {
        return "eggs"
    }

    override fun getName(): String {
        return super.getName()
    }    
}

File: Practica 8/Problema2/Dart/Bird.dart
import 'AbstractAnimal.dart';

class Bird extends AbstractAnimal {
  Bird(String name, int year) : super(name, year, 3);

  @override
  String move() {
    return "fly";
  }

  @override
  String breathe() {
    return "lungs";
  }

  @override
  String reproduce() {
    return "eggs";
  }
}
